agent/runtime-attach/build.gradle.kts

```diff
@@ -0,0 +1,14 @@
+plugins {
+  id("ai.java-conventions")
+}
+
+val otelContribAlphaVersion: String by project
+val otelVersion: String by project
+
+dependencies {
+  implementation("io.opentelemetry.contrib:opentelemetry-runtime-attach:$otelContribAlphaVersion")
+  implementation(project(":agent:agent", configuration = "shadow"))
+  testImplementation("io.opentelemetry:opentelemetry-extension-annotations:$otelVersion")
+  testImplementation("org.assertj:assertj-core:3.23.1")
+  testImplementation("org.junit.jupiter:junit-jupiter-api:.8.2")
```

Comment:
`.8.2` looks wrong? probably this version is covered by dependency management (and probably assertj also?)